fluentui_persona/src/main/java/com/microsoft/fluentui/persona/AvatarView.kt

```diff
@@ -148,6 +147,20 @@ open class AvatarView : AppCompatImageView {
             invalidate()
         }
 
+    /**
+     * Defines the [ContentDescription] applied to the avatar.
+     */
+    var avatarContentDescriptionLabel: String = ""
+        set(value) {
+            if (field == value)
```

Comment:
just a question, the repo doesn't apply a style requirement where all if statements require braces {}, even for single line cases?

Reply:
It is being followed at other places but not in these setvalue overrides across the app. Will have to take it up, in some other refactoring. Right now leaving it as it is